arcgispro/3D Data Co-Registration.pyt

```diff
@@ -353,27 +353,86 @@ class Register_MultiType(object):
         }
         codem_run_config = codem.CodemRunConfig(fnd_full_path, aoi_full_path, **kwargs)
         config = dataclasses.asdict(codem_run_config)
-
+        
+        #Add output to details pane with parameters and their values
+        arcpy.AddMessage("=============PARAMETERS=============")
+        for print_parameter in config:
+            arcpy.AddMessage(f"{print_parameter}={str(config[print_parameter])}")
```

Comment:
no need for the `str()` casting here; and since you're using both the key and value of the dictionary, you should iterate over `items`

```python
for parameter, value in config.items():
    arcpy.AddMessage(f"{parameter}={value}")
```